pkgs/tools/admin/dehydrated/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, bash, curl, openssl, makeWrapper, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "dehydrated-0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "lukas2511";
+    repo = "dehydrated";
+    rev = "ec49a4433b48a7bc8f178d06dad3f55cff24bdf3";
```

Comment:
Please create a new `version` attribute in the `mkDerivation` call and re-use it in `name` and for a tag in `rev` instead of a commit hash.